raiden_contracts/constants.py

```diff
@@ -11,6 +13,87 @@
 CONTRACT_MONITORING_SERVICE = 'MonitoringService'
 CONTRACT_RAIDEN_SERVICE_BUNDLE = 'RaidenServiceBundle'
 
+# Deployed contract information
+# Deployed to Ropsten revival on 2018-09-03 from
+# raiden-contracts@fc1c79329a165c738fc55c3505cf801cc79872e4
+ROPSTEN_TOKEN_NETWORK_REGISTRY_ADDRESS = '0xf2a175A52Bd3c815eD7500c765bA19652AB89B30'
```

Comment:
Why don't we group these contracts in a struct?

```python
class DeployedContract(typing.namedtuple):
    address: str
    deployed_at_block: int

class ContractsBundle(typing.namedtuple):
    version: str
    chain_id: ChainId
    token_network_registry_address: DeployedContract 
    secret_registry: DeployedContract
    endpoint_registry_address: DeployedContract

ROPSTEN = DeployedContracts(
    'Ropsten',
    ChainId.ROPSTEN,
    DeployedContract('0xf2a175A52Bd3c815eD7500c765bA19652AB89B30', 3604000),
    DeployedContract('0x16a25511A92C5ebfc6C30ad98F754e4c820c6822', 3604000),
    DeployedContract('0xEEADDC1667B6EBc7784721B123a6F669B69Eb9bD', 3604000),
)

ROPSTEN_LIMITED = DeployedContracts(
    'Ropsten - Upper limit deposits',
    ChainId.ROPSTEN,
    DeployedContract('0x6cC27CBF184B4177CD3c5D1a39a875aD07345eEb', 3604000),
    DeployedContract('0x8167a262Fa3Be92F05420675c3b409c64Be3d348', 3604000),
    DeployedContract('0xcF47EDF0D951c862ED9825F47075c15BEAf5Db1B', 3604000),
)
```

Then the mapping bellow `ID_TO_NETWORK_CONFIG` is not required, we can reference the smart contracts through the name of structure

Reply:
I wonder if the transaction hash would be useful information o have here too.